docs/notebooks/all_about_the_calendar.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>This is really awesome!!! Love the idea!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/lilio/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>